.circleci/scripts/python_doc_push_script.sh

```diff
@@ -71,7 +71,19 @@ cp -a ../vision/docs/source source/torchvision
 # Build the docs
 pip -q install -r requirements.txt || true
 if [ "$is_master_doc" = true ]; then
+  # export SPHINXOPTS="-WT --keep-going"
```

Comment:
Remove, or uncomment?

Reply:
This block, especially `lines-$(wc...`, is a little hard to read, a comment here would also be helpful.

Reply:
Adding a comment block and a TODO on the `SPHINXOPTS="-WT --keep-going"`, with a reference to gh-38011